src/backend/metal/src/command.rs

```diff
@@ -49,32 +49,51 @@ impl<'a> Deref for CommandBufferScope<'a> {
 
 impl QueuePool {
     fn find_queue(&mut self, device: &metal::DeviceRef) -> usize {
+        const POOL_SIZE: usize = 64;
```

Comment:
I guess we could use a much higher value here if we'd like

Reply:
Fair enough, at least in the future we should be able to safely lift it on macOS (any version) or iOS/tvOS >= 11.3 (11.3 contained the bug fix)